moduleframework/mtf_scheduler.py

```diff
@@ -33,24 +33,51 @@
 from moduleframework import common
 
 
-def cli():
-    file_name = os.path.basename(sys.argv[0])
+def mtfparser():
+    # for right name of man; man page generator needs it: script_name differs, its defined in setup.py
+    script_name = "mtf"
+    description = \
+"""
+VARIABLES
+
+       CONFIG defines the module configuration file. It defaults to config.yaml
+
+       MODULE defines tested module type, if default-module is not set in config.yaml.
+
+       URL to container.
+              E.g. URL=docker.io/modularitycontainers/haproxy if MODULE=docker
+
+       MODULEMDURL
+              overwrites the location of a moduleMD file.
+
+       and more at http://meta-test-family.readthedocs.io/en/latest/user_guide/environment_variables.html
```

Comment:
I would prefer to mention them.
`man` command is faster then looking at some web page.

Reply:
are we sure about that? when I saw that, man page seems too long to me, but I am not typical "manpage reader"